src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -59,7 +59,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
   UserService userService = UserServiceFactory.getUserService();
   if (!userService.isUserLoggedIn()) {
-   response.sendRedirect("/index.html");
+   response.sendRedirect("/home.jsp");
```

Comment:
This is a bit hard to notice, but if you redirect directly to home.jsp, you'll get a NullPointerException because home.jsp references some request attributes that do not exist. You need to redirect to HomeServlet first (using "/") to set those request attributes.